lib/enums/LoggerSeverity.js

```diff
@@ -0,0 +1,8 @@
+const LoggerSeverity = {
+    FATAL: 'fatal',
```

Comment:
fatal? I think it should be error, no?
either you only see errors or debug for more detailed log messages and not only errors

Reply:
"Fatal" and "Error" feel different to me, and right now "fatal" is what's in the spec: https://github.com/PerimeterX/px-enforcer-spec/wiki/Logger

I don't feel very strongly as long as it aligns with the spec. @shaul-badusa-px -- change this or change the spec?

Reply:
I talked with Shaul in the daily he also supports Error and not Fatal. also from inspecting the code where we use the "Fatal" logs it is not really fatal, means we are not terminating anything we just prompt an error, also the logger function itself called logger.error and not logger.fatal.

Reply:
👍